### PR TITLE
Run `npx ember-cli-update --from=3.17 --to=4.1.1` to sync addon blueprint

### PR DESCRIPTION
**Pull request by SergeAstapov**

### CODE REVIEW

**Comment by villander** [comment]
@SergeAstapov could you please add more details on the PR about migration to 4.1? Is there some break changes? What's the difference to update with `npx` instead `yarn`?

**Comment by villander** [comment]
@SergeAstapov there's a conflict here.

File: package.json [hunk 1]
```diff
@@ -85,5 +96,9 @@
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"
+  },
+  "volta": {
```
Comment (villander) [inline package.json:57]: We're not using volta, could you please remove it? or is it automatic from `ember-cli-update`?

File: tests/dummy/config/environment.js [hunk 2]
```diff
@@ -1,26 +1,26 @@
 'use strict';
 
-module.exports = function(environment) {
+module.exports = function (environment) {
   let ENV = {
     modulePrefix: 'dummy',
     environment,
     rootURL: '/',
-    locationType: 'auto',
+    locationType: 'history',
```
Comment (villander) [inline tests/dummy/config/environment.js]: Is it new from ember 4.x?

**Review by villander — COMMENTED** [review]
I have some questions, before move forward

File: package.json (hunk 1 above)
Comment (SergeAstapov) [reply package.json:57]: @villander removed

File: tests/dummy/config/environment.js (hunk 2 above)
Comment (SergeAstapov) [reply tests/dummy/config/environment.js]: @villander yeap, it is!

**Comment by villander** [comment]
@SergeAstapov any update here?

**Pull request closed without merging**